samples/static-server/src/main/java/okhttp3/sample/SampleServer.java

```diff
@@ -18,7 +18,7 @@
 import okio.Buffer;
 import okio.Okio;
 
-public class SampleServer extends Dispatcher {
+@SuppressWarnings("deprecation") public class SampleServer extends Dispatcher {
```

Comment:
no suppressions in sample code please

if we’re using deprecated things in sample code, that’s what we should fix.